src/main/java/movie/storage/model/dto/CinemaHallRequestDto.java

```diff
@@ -0,0 +1,9 @@
+package movie.storage.model.dto;
+
+import lombok.Data;
+
+@Data
+public class CinemaHallRequestDto {
+    private int capacity;
```

Comment:
you need to use Wrappers on this layer

Reply:
cause we don't need to have default primitive values in our fields

Reply:
Sure. Done.